starlette/responses.py

```diff
@@ -247,12 +255,13 @@ async def listen_for_disconnect(self, receive: Receive) -> None:
             if message["type"] == "http.disconnect":
                 break
 
-    async def stream_response(self, send: Send) -> None:
+    async def stream_response(self, send: Send, support_trailers: bool) -> None:
         await send(
             {
                 "type": "http.response.start",
                 "status": self.status_code,
                 "headers": self.raw_headers,
```

Comment:
Should we append the `Trailer` header? The key of the `trailers` attribute can be used to populate it... Or it's not Starlette's job?

Reference: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Trailer